README.md

```diff
@@ -108,7 +106,13 @@ IBEAM_PASSWORD=your_password123
 Run the following command:
 
 ```posh
-docker-compose up -d
+docker compose up -d
+```
+
+to check output
+
+```sh
+docker compose logs -f
```

Comment:
While this is a reasonable suggestion, I think we can go with the assumption that users will be familiar with how to read docker logs, hence this feels redundant

Reply:
yes coder will find out
but if we want to include more users then a hint won't harm
it's up to you to decide the target user group

Reply:
Yeah I see your point. So the way I see it is that I don't think our target group are people who don't know how to access docker logs. If one's missing this type of knowledge, one should look for more information on Docker first before using IBeam, seeing that it is primarily distributed as a Docker image. Hence, while you're right it won't harm, there needs to be a concrete reason for every addition other than 'it won't harm', and I think that isn't a case here and this is redundant